docs/Commands.md

```diff
@@ -116,7 +124,9 @@ Note: the numeric ID of an attachment can be found in the change log or in the l
 | Syntax      | `$ARISA_REMOVE_COMMENTS <username>` |
 | Permissions | Mod+                                |
 
-Finds all comments created by the user on a specific report and restricts them
+- `<username>`: Put the username of the user on Mojira as-is, including potential spaces. Do not encode spaces with the plus sign (`+`). Do not put quotes around it.
+
+Finds all comments created by the user on the current report and restricts them
```

Comment:
`on a specific report` is changed to `on the current report` here to make it more _specific_... I hope.

Reply:
Since this is already being changed, I think it would make sense to add a trailing period.